src/components/launch/SourceSelector.tsx

```
@@ -1,322 +1,370 @@
 }
 
-export function SourceSelector() {
+/**
+ * SourceSelectorContent - The actual list of sources
+ */
+export const SourceSelectorContent = ({
+	screenSources = [],
+	windowSources = [],
+	selectedSource = "Screen",
+	loading = false,
+	onSourceSelect = () => {},
```

Biome (2.4.14)

[error] Unexpected empty block.

(lint/suspicious/noEmptyBlockStatements)